container/features/src/main/resources/features-sentinel.xml

```diff
@@ -205,4 +205,27 @@
         <bundle>mvn:org.opennms.features.collection/org.opennms.features.collection.persistence.osgi/${project.version}</bundle>
         <bundle>mvn:org.opennms.features/org.opennms.features.newts/${project.version}</bundle>
     </feature>
+
+    <feature name="sentinel-distributed-cassandra-api"
+             description="OpenNMS :: Features :: Distributed :: Cassandra :: API"
+             version="${project.version}">
+        <bundle>mvn:org.opennms.features.distributed/org.opennms.features.distributed.cassandra.api/${project.version}</bundle>
+    </feature>
+    
+    <feature name="sentinel-newts-cassandra"
+             description="OpenNMS :: Features :: Distributed :: Newts Cassandra" version="${project.version}">
+        <feature>cassandra-driver</feature>
+        <feature>sentinel-newts</feature>
+        <feature>sentinel-distributed-cassandra-api</feature>
+        <bundle>mvn:org.opennms.features.distributed/org.opennms.features.distributed.cassandra.newts-impl/${project.version}</bundle>
+    </feature>
+
+    <feature name="sentinel-kvstore-cassandra"
+             description="OpenNMS :: Features :: Distributed :: Key Value Store :: Cassandra"
+             version="${project.version}">
+        <feature>sentinel-newts-cassandra</feature>
+        <feature>opennms-kvstore-api</feature>        
+        <bundle>mvn:org.opennms.features.distributed/org.opennms.features.distributed.kv-store.cassandra/${project.version}</bundle>
+        <bundle>mvn:org.opennms.features.distributed/org.opennms.features.distributed.kv-store.shell/${project.version}</bundle>
```

Comment:
Should we move the `shell` bundle into the `opennms-kvstore-api` feature instead? Seems like it would be useful to have for all impls.

Reply:
Yes, good point.